data/registry/otel-odddotnet.yml

```
@@ -0,0 +1,29 @@
+# cSpell:ignore Kenna odddotnet
+title: OddDotNet OTel Collector Test Harness
+registryType: utilities
+language: collector
+tags:
+  - proto
+  - protobuf
+  - testing
+  - odd
+  - observability
+  - collector
+  - test-harness
+  - odddotnet
+urls:
+  repo: https://github.com/OddDotNet/OddDotNet
+  docs: https://odddotnet.github.io/OddDotDocs/
+license: Apache 2.0
+description: OddDotNet is an OpenTelemetry Test Harness that works for any
+  language, built in .NET. It is a container image that accepts OpenTelemetry
+  signals over gRPC (and soon HTTP), and it includes a query language for
+  verifying and validating those signals.
```

```suggestion
description: OddDotNet is a Test Harness for OpenTelemetry that works for any
  language, built in .NET. It is a container image that accepts OpenTelemetry
  signals over gRPC, and it includes a query language for
  verifying and validating those signals.
```

Let's not make forward looking statements (and soon HTTP) in the registry entries. You can come back and update the description when the HTTP support is availab.e